android/src/main/java/com/pdftron/pdftronflutter/FlutterDocumentActivity.java

```diff
@@ -33,6 +33,8 @@
     private static AtomicReference<EventSink> sExportBookmarkEventEmitter = new AtomicReference<>();
     private static AtomicReference<EventSink> sDocumentLoadedEventEmitter = new AtomicReference<>();
 
+    private static Context context;
```

Comment:
static context is leak, remove, check again why this is needed

Reply:
In the implementation of setToolMode, 
boolean continuousAnnot = PdfViewCtrlSettingsManager.getContinuousAnnotationEdit(component.getContext());

ContinuousAnnotationEdit is not yet implemented, but will be in future PRs.

Reply:
this has to go, ask i said leak, change to a different way of doing, if unsure we can discuss